ui/js/group.js

```diff
@@ -160,6 +160,12 @@ require(['jquery', 'oae.core'], function($, oae) {
                         'title': oae.api.i18n.translate('__MSG__DISCUSSION__'),
                         'closeNav': true,
                         'class': 'oae-trigger-creatediscussion'
+                    },
+                    {
```

Comment:
It seems that this remains visible on the group pages even when no Jitsi server is set. Meetings should be disabled across board when there's no server in the config.